components/toggleButton/toggleButton.js

```diff
@@ -0,0 +1,94 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { getClassNamesWithMods } from '../_helpers';
+
+/**
+ * ToggleButton component.
+ */
+class ToggleButton extends Component {
+  /**
+   * Initialize default state
+   */
+  state = {
+    activeItem: 0,
+  }
+
+  /**
+   * Handlers onClick event
+   *
+   * @method handleSelectItem
+   * @param {Object} event
+   * @private
+   */
+  handleSelectItem = (event) => {
+    event.stopPropagation();
+
+    const index = this.props.items.indexOf(event.target.textContent);
+    this.setState({ activeItem: index });
+    (typeof this.props.handleSelect === 'function') && this.props.handleSelect(index);
+  }
+
+  /**
+   * Renders items markup
+   *
+   * @method renderItems
+   * @returns {JSX}
+   * @private
+   */
+  renderItems() {
+    return this.props.items.map((item, index) => {
+      const mods = {
+        'active': index === this.state.activeItem,
+      };
+      const classNameItem = getClassNamesWithMods('ui-toggle-button__item', mods);
+
+      return (
+        <li
+          className={classNameItem}
+          key={index}
+          onClick={this.handleSelectItem}
+        >
+          {item}
+        </li>
+      );
+    });
+  }
+
+  render() {
+    if (!(Array.isArray(this.props.items) && this.props.items.length > 1)) {
```

Comment:
Would be nice to destructure `items` from `this.props` before this line. It will give a bit more readability. And also using negation separately would simplify a bit.
```
const { items } = this.props;
if (!Array.isArray(items) || !items.length)) {
   ...
}
```
this way looks more readable for me.

Reply:
We even could remove check for array if use default value.

Reply:
Done.